src/components/ContentHighlights/ContentHighlightToast.jsx

```diff
@@ -0,0 +1,34 @@
+import { Toast } from '@edx/paragon';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const ContentHighlightToast = ({ toastText }) => {
+  /* Toast visibility state initially set to false to ensure the toast's
+  fade-in animation occurs on mount once `showToast` set to true. */
+  const [showToast, isToastShown] = useState(false);
+  const handleClose = () => {
+    isToastShown(false);
+  };
+  useEffect(() => {
+    isToastShown(true);
+  }, []);
+  return (
+    <Toast
+      onClose={() => handleClose()}
+      show={showToast}
+    >
+      {toastText}
+    </Toast>
+
+  );
+};
+
+ContentHighlightToast.propTypes = {
+  toastText: PropTypes.string,
+};
+
+ContentHighlightToast.defaultProps = {
+  toastText: '',
```

Comment:
[curious] Under what situations will `toastText` be an empty string here? Why do we attempt to render a `Toast` when there is no text associated with it considering `toastText` is optional here?